app/jobs/after_transition_tasks_for_rejected_return_job.rb

```diff
@@ -2,7 +2,7 @@ class AfterTransitionTasksForRejectedReturnJob < ApplicationJob
   def perform(submission, transition)
     transition ||= submission.last_transition
 
-    # submission.tax_return&.transition_to(:file_rejected) #ctc only
+    submission.tax_return&.transition_to(:file_rejected) if submission.is_for_federal_filing?
```

Comment:
Is this going to also affect gyr?

Reply:
no, GYR doesn't do efiling -- this would only impact CTC. I had commented this out a few weeks ago but decided to add this ctc check instead. We aren't using this code rn but unclear if we should keep it all in tack